pyairtable/api/abstract.py

```diff
@@ -16,25 +16,28 @@
 
 class ApiAbstract(metaclass=abc.ABCMeta):
     VERSION = "v0"
-    API_BASE_URL = "https://api.airtable.com/"
     API_LIMIT = 1.0 / 5  # 5 per second
-    API_URL = posixpath.join(API_BASE_URL, VERSION)
     MAX_RECORDS_PER_REQUEST = 10
 
     session: Session
+    endpoint_url: str
+    api_url: str
     tiemout: TimeoutTuple
 
     def __init__(
         self,
         api_key: str,
         timeout: Optional[TimeoutTuple] = None,
         retry_strategy: Optional[Retry] = None,
+        endpoint_url: Optional[str] = "https://api.airtable.com",
     ):
         if not retry_strategy:
             self.session = Session()
         else:
             self.session = _RetryingSession(retry_strategy)
 
+        self.endpoint_url = endpoint_url
+        self.api_url = posixpath.join(self.endpoint_url, self.VERSION)    
```

Comment:
I might just combine these two; will we ever need the endpoint _without_ the version number? Would we be just as well off removing one of them and just storing the combined URL?

Reply:
https://github.com/gtalarico/pyairtable/pull/243/commits/8eeaa44204ac2d058676b8166fcdd8de1ffa8fb2